libs/sdk-ui-kit/src/List/InsightListItem.tsx

```diff
@@ -0,0 +1,145 @@
+// (C) 2007-2020 GoodData Corporation
+import React, { Component, createRef } from "react";
+import cx from "classnames";
+import { FormattedMessage } from "react-intl";
+
+import { stringUtils } from "@gooddata/util";
+import ShortenedText from "@gooddata/goodstrap/lib/core/ShortenedText";
+import { getDateTimeConfig } from "@gooddata/goodstrap/lib/data/date";
+
+import { InsightListItemDate } from "./InsightListItemDate";
+import { Button } from "../Button";
+
+const VISUALIZATION_TYPE_UNKNOWN = "unknown";
+const WIDGET_TYPE_KPI = "kpi";
+
+const visualizationIconWidthAndPadding = 42;
+
+const tooltipAlignPoints = [
+    {
+        align: "cr cl",
+    },
+    {
+        align: "cl cr",
+        offset: {
+            x: -visualizationIconWidthAndPadding,
+            y: 0,
+        },
+    },
+];
+
+/**
+ * @internal
+ */
+export interface IInsightListItemProps {
+    isLoading?: boolean;
+    isLocked?: boolean;
+    isSelected?: boolean;
+
+    title?: string;
+    updated?: string;
+    type?: string;
+    width?: number;
+
+    onClick?: () => void;
+    onDelete?: () => void;
+}
+
+/**
+ * @internal
+ */
+export class InsightListItem extends Component<IInsightListItemProps> {
+    private shortenedTextRef = createRef<ShortenedText>();
+
+    public render(): JSX.Element {
+        const {
+            title,
+            updated,
+            type = VISUALIZATION_TYPE_UNKNOWN,
+            isSelected,
+            isLoading,
+            onClick,
+        } = this.props;
+
+        const iconClassName = cx("gd-vis-type", `gd-vis-type-${type}`);
+
+        const visualizationListItemClassname = cx(
+            "gd-visualizations-list-item",
+            `s-${stringUtils.simplifyText(title)}`,
+            {
+                "is-selected": isSelected,
+            },
+        );
+
+        return (
+            <div className={visualizationListItemClassname} onClick={onClick}>
+                <div className={iconClassName} />
+                <div className="gd-visualizations-list-item-content">
+                    <div className="gd-visualizations-list-item-content-name">
+                        {this.renderLock()}
+                        <ShortenedText ref={this.shortenedTextRef} tooltipAlignPoints={tooltipAlignPoints}>
+                            {isLoading ? <FormattedMessage id="gs.visualizationsList.loading" /> : title}
+                        </ShortenedText>
+                    </div>
+                    <div className="gd-visualizations-list-item-content-date">
+                        {this.renderUpdatedDateTime(updated)}
+                    </div>
+                </div>
+                {this.renderActions()}
+            </div>
+        );
+    }
+
+    public componentDidUpdate(prevProps: IInsightListItemProps): void {
+        if (prevProps.width !== this.props.width && this.shortenedTextRef.current) {
+            // TODO INE remove any when GD has correct typings
```

Comment:
Is this actionable now after the Goodstrap 69 release?

Reply:
Unluckily this issue was not fixed in new GS :(